autonomi/src/client/data.rs

```diff
@@ -218,26 +220,33 @@ impl Client {
         info!("Getting chunk: {addr:?}");
 
         let key = NetworkAddress::from_chunk_address(ChunkAddress::new(addr)).to_record_key();
-
+        debug!("Converted chunk address to network record key: {:?}", key);
```

Comment:
This is trivial and cannot fail too. It is not required to tell that we've obtained the key. Instead the `key` can be used in other places to track a particular data on the network. For eg., `debug!("Record fetched from network: {:?}", key);`